tembo-operator/src/cloudnativepg/cnpg.rs

```diff
@@ -877,11 +877,6 @@ async fn pods_to_fence(cdb: &CoreDB, ctx: Arc<Context>) -> Result<Vec<String>, A
             .as_ref()
             .and_then(|s| s.first_recoverability_time.as_ref())
             .is_none()
-        && cdb
-            .status
-            .as_ref()
-            .and_then(|s| s.last_fully_reconciled_at.as_ref())
-            .is_none()
```

Comment:
@nhudson WDYT about this change

Reply:
Since you're removing it from the `CoreDB` API, the `CoreDB` validation will not be able to complete.  So for each one it reads it will throw an error.